Plugin/AddTjSyncDateToGrid.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Taxjar\SalesTax\Plugin;
+
+use Magento\Framework\View\Element\UiComponent\DataProvider\CollectionFactory;
+use Magento\Sales\Model\ResourceModel\Order\Grid\Collection as OrderGridCollection;
+use Magento\Sales\Model\ResourceModel\Order\Creditmemo\Grid\Collection as CreditmemoGridCollection;
+
+class AddTjSyncDateToGrid
+{
+    /**
+     * Join tj_salestax_sync_date in the order and creditmemo admin grids
+     *
+     * @param CollectionFactory $subject
+     * @param $collection
+     * @return \Magento\Framework\Data\Collection;
+     */
+    public function afterGetReport(
+        CollectionFactory $subject,
+        $collection
+    ) {
+        if ($collection instanceof OrderGridCollection) {
```

Comment:
It looks like there is an overload of this method that provides the request name that we could use instead of `instanceof`. This might be slightly more preferred?

https://store.magenest.com/blog/add-columns-in-sales-order-grid-magento-2/

Reply:
Magento uses php's splat operator to pass arguments as the 3rd variable in the 'after' plugins.  I think it will be more reliable to continue checking the type of collection, since 3rd party extensions could modify the arguments being passed. 

https://github.com/magento/magento2/blob/f824b9eea6d35f7e095e1471a0b92a871efc1f0b/lib/internal/Magento/Framework/Interception/Interceptor.php#L146